treeherder/model/derived/jobs.py

```diff
@@ -431,6 +431,20 @@ def get_performance_series_summary(self, interval_seconds):
 
         return series_summary
 
+    def get_platforms(self, interval_seconds):
```

Comment:
I meant this specific method should be renamed to `get_performance_platforms`, not the web service method. :)

So change this and revert the changes there. The behaviour you had before was good (/performance-data/platforms/ is a good endpoint name, as the performance-data part provides enough context to figure out what it's about)